### DISCUSSION TITLE
Buttons become unavailable after pressing FC Start

### DISCUSSION THREAD

**Discussion by rchiker05** [artisan-roaster-scope/artisan#1585] [Q&A]
Good afternoon. 

I am running version 2.10.4 on my Windows 10 laptop and am having an issue with the buttons becoming unavailable after I press the FS Start button. The graph continues to work but I can't press the FC End button or the SC start etc. They are all just greyed out. 

Am I doing something wrong or have something misconfigured maybe? This feels like a bug to me but wanted to ask for some guidance here first before submitting anything. 

Thanks

**Comment by rchiker05** [comment]
Actually just did the same thing after pressing the Dry End button. I couldn't do anything else with the buttons until I reset and started the roast over in Artisan.

**Reply by poundy** [reply]
the buttons work fine on my install, I've been using 2.10.4 in my home roasting without an issue, so it's something you've set up or the way you're doing what you're doing. 
You will need to attach your settings here so that someone can load them and test. Will probably be worth posting a quick video somewhere to show your process as well.
You might also want to describe background - you posted not that long ago about a meter not working, so is your setup a fresh install and are you a first-time user of Artisan?

**Reply by rchiker05** [reply]
Thanks for the response. I am a new user and have watched some setup videos and read through setup material. 

I have everything shutdown now but will review everything again and see if I missed something. My setup is very basic, only 2 probes from the Mastech MS6514 and that is the profile I have selected. No custom buttons or anything like that. 

Anyway, I will review again and can post my configuration when I get back to my computer and have it running again. 

Thanks

**Reply by poundy** [reply]
ok, thought so. One thing to be clear, Artisan will lead you through a roast, by only permitting you to hit milestone events "in order", and the more i think about your situation it seems to me that you've likely missed marking a critical roasting event prior to pressing either of these two event buttons. Did you forget to mark "Charge" by any chance? I don't know the code logic behind what events are mandatory but Charge is the true "start" of your coffee's roast, but missing Charge may well be the critical one you're missing. From there, if you next hit DE or FCS would not be unusual (although missing DE means you have less info on how you set up for your maillard phase)

**Reply by rchiker05** [reply]
That’ll make sense. 

I don’t think that’s what’s happening in my case though. On this last roast, I hit start, brought my drum to temp, dumped the beans in and hit Charge. Let it go through the dry cycle hit the DE button. Then let it go to first crack I hit the FC button. After that, all the buttons are grayed out and I can’t use any of them.

**Reply by rchiker05** [reply]
Here is a copy of my settings I am using. I reviewed the setup documentation again today as I was capturing screen shots of all of these. The only thing I found in the review was the sampling time was set to 2S instead of 3S as the documentation stated. However I had never changed this setting so 2S was the default setting from the installation. The documentation states the default is 3S so I updated that to match the documentation. 

I am still playing around with the roast phases. I have a glass drum roaster and roast in my unfinished garage. It works well overall and I enjoy being able to see the stages of the roast and smells that go along with it. It's definitely a more manual way to roast since nothing is automated and environmental conditions are variable. I have been able to get some really good roasts and am definitely hooked on this hobby. I am trying to up my game a bit with Artisan. 

I haven't tried another roast yet, but wanted to share my settings to see if you notice anything that might cause the issue I described. Next roast I can take a video of the behavior with the buttons. 

[ArtisanSettings.pdf](https://github.com/user-attachments/files/15848749/ArtisanSettings.pdf)
